src/ch/ehi/ili2db/fromxtf/FromXtfRecordConverter.java

```diff
@@ -74,6 +74,7 @@
 	private HashMap tag2class=null;
 	private Integer defaultEpsgCode=null;
     private Map<AttributeDef,EnumValueMap> enumCache=new HashMap<AttributeDef,EnumValueMap>();
+    private Map<String, String> displayNameCache = new HashMap<String, String>();
```

Comment:
Das kann so nicht funktionieren (weil verschiedene Aufzähltypen die selben Aufzählwerte enthalten können). Vorschlag: die Abbildung ili-name->dispName in die EnumValueMap integrieren (dann löst sich das Problem von selbst...)

Reply:
Ja merci, und ein Test war falsch wegen die selbe Aufzählwerte in attr2 und attr3.
Jetzt ist fixed:
https://github.com/claeis/ili2db/pull/415/commits/614e59069d1f69371db10dcaa826ddb1cb7be0d5

Reply:
liegt wahrscheinlich an einem veralteten Enum.ili im $HOME/.ilicache/$DBURL/Enum.ili das beim Daten-Import benutzt wird.

Reply:
Merci, ich habe die cached Enum.ili gelöscht und es hat geklappt!

Jetzt ich habe alle Änderungen implementiert gemäss deine Kommentare.

Kann Mann irgendwie die cache deaktivieren ? Ist praktisch während die Entwicklung.

Reply:
Nein, kann man aktuell nicht (Ich bin zwar auch schon darüber gestolpert, aber meistens ändert man ja das Modell nicht)